doc/ReleaseNotes.md

```diff
@@ -1,5 +1,14 @@
 ﻿### 3.5.4 / 4.0.4-beta1
 * ICodeInspector API is obsolete.
+* ViewBag data can now be used in overwritten SetModel calls
+  Viewbag is set like the model on template creation (not in the ExecuteContext)
+  This leads to some minor breaking changes
+  - Creating a ExecuteContext with a non-null viewbag throws a NotSupportedException (obsolete API)
+  - added a DynamicViewBag parameter to IInternalTemplateService.Resolve (you should not need to use that API)
+  - ITemplate.SetModel has changed to ITemplate.SetData (you are not affected if you use TemplateBase or TemplateBase<T> as base class, which is recommended)
+  
+While these changes are (strictly speaking) breaking they should hardly affect anyone, so this update will get only a minor version jump.
+
```

Comment:
According to semver.org, breaking changes require incrementing the major version, no matter how many people are affected by the break. It is the only way to reliably let people know that something major changed. A user should be able to install a point release with confidence that it will not break their app.